templates/app.yaml

```diff
@@ -149,6 +153,9 @@ Resources:
         - Namespace: 'aws:autoscaling:launchconfiguration'
           OptionName: EC2KeyName
           Value: !Ref EC2KeyName
+        - Namespace: 'aws:autoscaling:launchconfiguration'
+          OptionName: SSHSourceRestriction
+          Value: !Sub 'tcp, 22, 22, ${VpnCidr}'
```

Comment:
This is an AWS beanstalk setup.  All EC2 instances are in a private subnet behind a load balancer.  Access is already restricted by the load balancer therefore I don't think this is necessary.

Reply:
This might not be necessary, but it's easy to add a second layer of security and I'd rather prevent the related CIS finding than suppress it.